charts/gcp/gcp-vm-instance-stop/engine.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: litmuschaos.io/v1alpha1
+kind: ChaosEngine
+metadata:
+  name: gcp-vm-chaos
+spec:
+  engineState: 'active'
+  chaosServiceAccount: gcp-vm-instance-stop-sa
+  jobCleanUpPolicy: 'retain'
```

Comment:
can we remove the job cleanup policy? It is by default retain. It is just to keep it uniform.

Reply:
Ofcourse, fixed now